packages/disclaimer/src/lib/Disclaimer.tsx

```diff
@@ -8,13 +9,15 @@ type AvDisclaimerProps = {
   accent?: boolean;
   /** If true, the Disclaimer displays with less padding  */
   dense?: boolean;
+  /** The id of the element */
+  id?: string;
 };
 
 export interface DisclaimerProps extends AvDisclaimerProps {
   /** The text to display in the header */
   headerText?: string;
   /** The text to display in the body */
-  description: string;
+  description: string | ReactNode;
```

Comment:
`string` is included in `ReactNode` so we should be able to just use `ReactNode` unless we specifically want to point out `string`.

Reply:
fixed